source/OutfitInfoDisplay.cpp

```diff
@@ -560,7 +565,13 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 			if(values[i])
 			{
 				attributeLabels.emplace_back(VALUE_NAMES[i].first + PER_SECOND);
-				attributeValues.emplace_back(Format::Number(60. * values[i] / reload) + VALUE_NAMES[i].second);
+				if(fullDropoff != 1 && VALUE_NAMES[i].first.find("damage") != string::npos)
+				{
+					attributeValues.emplace_back(Format::Number(60. * values[i] / reload) + VALUE_NAMES[i].second
+						+ " - " + Format::Number(60. * fullDropoff * values[i] / reload) + VALUE_NAMES[i].second);
+				}
+				else
+					attributeValues.emplace_back(Format::Number(60. * values[i] / reload) + VALUE_NAMES[i].second);
```

Comment:
```suggestion
				attributeValues.emplace_back(Format::Number(60. * values[i] / reload) + VALUE_NAMES[i].second);
				if(fullDropoff != 1 && VALUE_NAMES[i].first.find("damage") != string::npos)
					attributesValues.back() += " - " + Format::Number(60. * fullDropoff * values[i] / reload) + VALUE_NAMES[i].second);
```
I think this works? not sure though

we also may as well restructure a bit and put this in a namespace function (the whole if(reload) thingy but I'm not sure if that's better or a lambda (or if the small replication is fine, mb)

Reply:
```suggestion
				attributeValues.emplace_back(Format::Number(60. * values[i] / reload) + VALUE_NAMES[i].second);
				if(fullDropoff != 1 && VALUE_NAMES[i].first.find("damage") != string::npos)
					attributesValues.emplace_back(" - " + Format::Number(60. * fullDropoff * values[i] / reload) + VALUE_NAMES[i].second);
```
consistency?

Reply:
Implemented in both places.